frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Form,
+  TextInputTypes,
+  ActionGroup,
+  Alert,
+  Button,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { FormikProps, FormikValues } from 'formik';
+import { ButtonBar } from '@console/internal/components/utils';
+import { MultiColumnField, InputField, DropdownField } from '../formik-fields';
+
+enum accessRoles {
+  '' = 'Select a role',
+  admin = 'Admin',
+  view = 'View',
+}
+
+const ProjectAccess: React.FC<FormikProps<FormikValues>> = ({
+  handleSubmit,
+  handleReset,
+  isSubmitting,
+  status,
+  errors,
+  dirty,
+}) => {
+  return (
+    <Form onSubmit={handleSubmit}>
+      <div className="co-m-pane__form">
```

Comment:
Is this necessary? I don't see it present in the import forms but I do see it in the pipeline tab forms.

Reply:
I think this is required. The various sections in the import form use `FormSection` component which adds `co-m-pane__form` property based on a condition.

Reply:
Try without and let me know. It could very well be required (for now).

Reply:
@christianvogt  Below screenshot is without the `co-m-pane__form` property 

![Screenshot from 2019-10-09 23-19-25](https://user-images.githubusercontent.com/22490998/66506617-52455380-eaeb-11e9-9c7d-8bc4f3f217bb.png)